pubspec.yaml

```
@@ -41,9 +41,9 @@ dev_dependencies:
   over_react_test: ^2.5.0
   test: ^1.0.0
 
-
 dependency_overrides:
   react:
     git:
-      url: https://github.com/cleandart/react-dart.git
-      ref: 5.1.0-wip
+      url: git@github.com:cleandart/react-dart.git
```

```suggestion
      url: https://github.com/cleandart/react-dart.git
```
This should get CI working again